app/upload/directives/audio-version.component.ts

```diff
@@ -0,0 +1,96 @@
+import {Component, Input, OnInit, OnDestroy} from 'angular2/core';
+import {Subscription} from 'rxjs';
+import {UploadService} from '../services/upload.service';
+import {UploadFileSelect} from './upload-file-select.directive';
+import {AudioFileComponent} from './audio-file.component';
+import {HalDoc} from '../../shared/cms/haldoc';
+import {AudioModel} from '../models/audio.model';
+
+// TODO: why can't atom find this? jspm bundles it just fine
+import {Dragula, DragulaService} from 'ng2-dragula';
```

Comment:
For the life of me, I couldn't make Atom happy with this line.  And the `ng2-dragula` module definitely has a typescript def file, so not sure what's going on.